### PR TITLE
Minor refactor for javascript

### PR DESCRIPTION
**Pull request by Ragg-**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
 'use restrict';
-document.addEventListener('DOMContentLoaded', function (event) {
-  var sourceUri = 'https://www.pixiv.net/ranking.php?mode=daily&format=json&content=illust';
-  var promiseList = [];
+document.addEventListener('DOMContentLoaded', (event) => {
```
Comment (notherno) [inline main.js]: `event => {`

File: main.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  const SOURCE_URI = 'https://www.pixiv.net/ranking.php?mode=daily&format=json&content=illust';
+  const promiseList = [];
 
-  for (var i = 1; i <= 3; i++) {
-    var p = axios.get(sourceUri + '&p=' + i);
+  for (let i = 1; i <= 3; i++) {
+    const p = axios.get(SOURCE_URI + '&p=' + i);
```
Comment (notherno) [inline main.js]: Why not use template literal?

File: main.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
     promiseList.push(p);
   }
 
-  Promise.all(promiseList).then(function (values) {
-    var items = [];
-    var illustrations = [];
+  Promise.all(promiseList).then((values) => {
```
Comment (notherno) [inline main.js]: `values => {`

File: main.js [hunk 4]
```diff
@@ -54,34 +54,34 @@ document.addEventListener('DOMContentLoaded', function (event) {
...
   img.src = imageUrl;
   img.alt = author + ' / ' + title;
   return img;
 }
 
 function createGalleryItem(illustrationId, imageUrl, title, author) {
-    var linkUrl = 'https://www.pixiv.net/i/' + illustrationId;
-    var img = createIllustrationElement(imageUrl, title, author);
-    var anchor = document.createElement('a');
+    const linkUrl = 'https://www.pixiv.net/i/' + illustrationId;
```
Comment (notherno) [inline main.js]: Can be replaced with template literal?

File: main.js [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
 
-  Promise.all(promiseList).then(function (values) {
-    var items = [];
-    var illustrations = [];
+  Promise.all(promiseList).then((values) => {
+    const items = [];
+    let illustrations = [];
 
-    values.forEach(function (response) {
+    values.forEach((response) => {
```
Comment (notherno) [inline main.js]: `response => {`

File: main.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
       if (response.status === 200) {
         // Extracting illustration info
-        response.data.contents.forEach(function (content) {
+        response.data.contents.forEach((content) => {
```
Comment (notherno) [inline main.js]: `content => {`

**Comment by tamanobi** [comment]
I'm regacy JavaScripter. Thanks modern JavaScripter.

File: main.js (hunk 2 above)
Comment (Ragg-) [inline main.js]: :sori:

File: main.js (hunk 1 above)
Comment (Ragg-) [inline main.js]: religious circumstances

File: main.js (hunk 4 above)
Comment (tamanobi) [inline main.js]: What is template literal?

Comment (notherno) [inline main.js]:
```
`https://www.pixiv.net/i/${illustrationId}`
```

File: main.js (hunk 1 above)
Comment (tamanobi) [inline main.js]: I think so too. I wanna reject it.

File: main.js (hunk 4 above)
Comment (tamanobi) [inline main.js]: Sorry, I don't know its syntax. It's so cool.

**Comment by tamanobi** [comment]
@notherno Can you make new merge requests to fix single argument arrow function and change string concatnation to template literal?

File: main.js (hunk 4 above)
Comment (fsubal) [inline main.js]: `something ${like} ${this}`

**Pull request merged**